### DISCUSSION TITLE
Nano 33 IoT (SAMD21) smooth fonts question

### DISCUSSION THREAD

**Discussion by deruiter** [Bodmer/TFT_eSPI#1427] [Q&A - Fonts] [answered]
First thank you for your amazing library!  

Q: what is the best way for my board (Arduino Nano-33-Iot > SAMD21) to use smooth fonts? 
Do I need to use the FLASH-ARRAY method or can I better use LittleFS?  

Sorry for my lack of knowledge, I just managed to get some characters displayed using platformio with all the settings involved.

Checking size .pio/build/nano_33_iot/firmware.elf
Advanced Memory Usage is available via "PlatformIO Home > Project Inspect"
RAM:   [=         ]   8.2% (used 2692 bytes from 32768 bytes)
Flash: [==        ]  15.3% (used 40200 bytes from 262144 bytes)
================================================ [SUCCESS] Took 0.61 seconds

**Comment by Bodmer** [comment, marked as answer]
FLASH arrays are slightly faster normally as there is no filing system overhead. But less convenient as the vlw files have to be converted to an array.  You need to be careful as you only have 32kbytes of RAM and so use small fonts and or use very few characters in a font as when it is loaded with loadFont(..) a copy of the font parameters/metrics is stored in RAM for faster access.
